examples/cisco/demo_ospf.pp

```diff
@@ -86,4 +86,19 @@
     stub            => true,
     stub_no_summary => false,
   }
+
+  cisco_ospf_area { 'dark_blue vrf2 5000':
+    ensure                 => 'present',
+    authentication         => 'clear_text',
+    default_cost           => 4444,
+    filter_list_in         => 'bbb',
+    filter_list_out        => 'ccc',
+    nssa                   => true,
+    nssa_default_originate => true,
+    nssa_no_redistribution => true,
+    nssa_no_summary        => true,
+    nssa_route_map         => 'aaa',
+    nssa_translate_type7   => 'supress_fa',
+    range                  => [['10.3.0.0/16', '234'], ['10.3.1.1/32', 'not_advertise']],
+  }
```

Comment:
IMO, we should keep these examples simple. You've already got examples above that show how to use authentication, cost, filter_list, range, etc so it just clutters up the nssa example by repeating them. I would just do this:

```
cisco_ospf_area { 'dark_blue my_nssa_vrf 5000':
    ensure                 => 'present',
    nssa                   => true,
    nssa_default_originate => true,
    nssa_no_redistribution => true,
    nssa_no_summary        => true,
    nssa_route_map         => 'aaa',
    nssa_translate_type7   => 'supress_fa',
}
```